### ISSUE TITLE
Pulls in EOL extension org.gnome.Crosswords.PuzzleSets.puzzlepull

### ISSUE THREAD

**Issue by roshanshariff** [flathub/org.gnome.Crosswords#6]
Installing this package brings in the flatpak extension `org.gnome.Crosswords.PuzzleSets.puzzlepull`, and thereafter Flatpak repeatedly warns about it being end-of-life.

See flathub/org.gnome.Crosswords.PuzzleSets.puzzlepull#3

**Comment by bbhtt** [comment]
That extension is supposed to be replaced by this https://github.com/flathub/org.gnome.Crosswords.PuzzleSets.xword-dl 

This sounds like https://github.com/flatpak/flatpak/issues/5177. Crosswords doesn't depend on any specific extension.

Do this: 

Remove any unused apps/runtime: `flatpak uninstall --unused`.
Remove that specific extension: `flatpak remove org.gnome.Crosswords.PuzzleSets.puzzlepull`.
Then mask it so that it is never installed or updated: `flatpak mask [--user] org.gnome.Crosswords.PuzzleSets.puzzlepull`. 

Now if you install or update `org.gnome.Crosswords`, that extension won't be pulled in.

**Comment by roshanshariff** [comment]
@bbhtt, thank you very much! Masking out the extension works perfectly until that flatpak bug gets fixed.

**Comment by LSeelig** [comment]
Would the fix be in this repo or upstream?

**Comment by blandford** [comment]
Oh hey! I never check this repo for issues and just noticed this old one. It looks like it's still a problem. @bbhtt is there anything I can do on my end to fix it. Any idea if flathub can be fixed here?

**Comment by bbhtt** [comment]
This is not an issue in Flathub but Flatpak as I linked the bug already.

**Comment by blandford** [comment]
Yeah, sorry. I meant flatpak. Just wondering if I can work around it in my manifest/

**Comment by bbhtt** [comment]
I can't think of a non intrusive way. One way is to change the version here to say `2` https://github.com/flathub/org.gnome.Crosswords/blob/fa67f02d8e72991f4151a82bd8e61599657fb5a7/org.gnome.Crosswords.json#L51C13-L51C29

then `org.gnome.Crosswords.PuzzleSets.puzzlepull` won't be installed as it won't have a `2` branch. But this requires all the other extensions to change their respective branches to `2` as well.

The other solution is I can remove the EOL from `org.gnome.Crosswords.PuzzleSets.puzzlepull` and make it "empty". It will still be installed, but it won't have any contents in it and the EOL warning won't be there. Then I can archive the repo again.
